resources/windows/wix/project.wxs.erb

```diff
@@ -25,7 +25,9 @@
       <!-- We can add all components by referencing this one thing -->
       <ComponentGroupRef Id="ProductComponentGroup" />
       <ComponentGroupRef Id="RegistryComponentGroup" />
-      <ComponentGroupRef Id="ServiceListGroup" />
+      <%- get_services.each do |service| -%>
+      <ComponentGroupRef Id="<%= service.component_group %>" />
```

Comment:
If we have an ID listed here, but not a corresponding ID in a service file, will we get an error?